wodles/vuls/vuls.py

```diff
@@ -0,0 +1,295 @@
+    elif type == 'redhat' or type == 'centos':
+        source = 'RedHat OVAL'
+    elif type == 'ubuntu':
+            source = 'Ubuntu OVAL'
+    elif type == 'debian':
+            source = 'Debian OVAL'
+    elif type == 'oracle':
+            source = 'Oracle OVAL'
+    else:
+        print('Error: Invalid {0} type.'.format(type))
+        sys.exit(1)
+
+    link = cve['CveContents'][type]['SourceLink']
+    last_modified = cve['CveContents'][type]['LastModified']
+    return source, link, last_modified
+
+def extract_CVEscore(cve, type):
+    cvss2 = cve['CveContents'][type]['Cvss2Score']
+    cvss3 = cve['CveContents'][type]['Cvss3Score']
+    score = cvss2 if cvss2 > cvss3 else cvss3
+    return score
+
+def has_vector(cve, type):
+    return type in cve['CveContents']
+
+def change_vector(type, family):
+    return family if type == 'nvd' else 'nvd'
+
+def send_msg(wazuh_queue, header, msg):
+    msg['integration'] = 'vuls'
+    debug(json.dumps(msg, indent=4))
+    msg = '{0}{1}'.format(header, json.dumps(msg))
+    s = socket(AF_UNIX, SOCK_DGRAM)
+    s.connect(wazuh_queue)
+    s.send(msg.encode())
+    s.close()
+
+def debug(msg):
+    if enable_debug:
+        print(msg)
+
+def update_oval(OS, update_os_version):
+    global oval_db_fetcher
+    global vuls_log
+    global vuls_path
+
+    if not update_os_version:
+        print('Error: To update the OVAL database, the OS version must be attached with --os-version. You can do it automatically with --autoupdate.')
+        sys.exit(1)
+    debug('Updating {0} {1} OVAL database...'.format(OS, update_os_version))
+    call([oval_db_fetcher, 'fetch-{0}'.format(OS), '-dbpath={0}/oval.sqlite3'.format(vuls_path), '-log-dir={0}'.format(vuls_log), update_os_version])
+
+def update(update_nvd, update_rh, update_ub, update_deb, update_orac, os_name, update_os_version):
+    if update_nvd:
+        debug('Updating NVD database...')
+        for i in range(nvd_year, (int(str(datetime.now()).split('-')[0]) + 1)):
+            call([cve_db_fetcher, 'fetchnvd', '-dbpath={0}/cve.sqlite3'.format(vuls_path), '-log-dir={0}'.format(vuls_log), '-years', str(i)])
+
+    if update_rh or os_name == 'redhat':
+        debug('Updating Redhat OVAL database...')
+        update_oval('redhat', update_os_version) #5 6 7
+    elif update_ub or os_name == 'ubuntu':
+        debug('Updating Ubuntu OVAL database...')
+        update_oval('ubuntu', update_os_version) #12 14 16
+    elif update_deb or os_name == 'debian':
+        debug('Updating Debian OVAL database...')
+        update_oval('debian', update_os_version) #7 8 9 10
+    elif update_orac or os_name == 'oracle':
+        debug('Updating Oracle OVAL database...')
+        update_oval('oracle', update_os_version) #5 6 7
+
+def main(argv):
+
+    # Minimum CVSS for reporting
+    cvss_min = 0
+    # CVSS source
+    cvss_source=''
+    # Message header
+    header = '1:Wazuh-VULS:'
+    # Notify message header
+    notify_header = '9:Wazuh-VULS:'
```

Comment:
The location for Rootcheck events must be `rootcheck`. This will prevent the manager from creating files like `/var/ossec/queue/rootcheck/Wazuh-VULS`. The file must always be the name and IP of the agent, or `rootcheck`. Remoted will deal with agent names.

The line should be:
```python
notify_header = '9:rootcheck:'
```